crates/pallet-domains/src/lib.rs

```diff
@@ -1918,9 +1918,19 @@ mod pallet {
 
                 let timestamp = T::BlockTimestamp::block_timestamp();
 
+                let transaction_byte_fee: Balance = T::StorageFee::transaction_byte_fee()
+                    .try_into()
+                    .map_err(|_| ())
```

Comment:
Look like an unnecessary `map_err` as the next line is `expect`

Reply:
The error type isn't constrained to implement `Debug`, so we can’t call `expect()` on it.